plugins/inputs/gnmi/handler.go

```diff
@@ -98,21 +116,55 @@ func (h *handler) subscribeGNMI(ctx context.Context, acc telegraf.Accumulator, t
 				h.log.Debugf("update_%v: %s", t, string(buf))
 			}
 		}
-
 		if response, ok := reply.Response.(*gnmiLib.SubscribeResponse_Update); ok {
-			h.handleSubscribeResponseUpdate(acc, response)
+			h.handleSubscribeResponseUpdate(acc, response, reply.GetExtension())
 		}
 	}
 	return nil
 }
 
 // Handle SubscribeResponse_Update message from gNMI and parse contained telemetry data
-func (h *handler) handleSubscribeResponseUpdate(acc telegraf.Accumulator, response *gnmiLib.SubscribeResponse_Update) {
+func (h *handler) handleSubscribeResponseUpdate(acc telegraf.Accumulator, response *gnmiLib.SubscribeResponse_Update, extension []*gnmiExt.Extension) {
 	var prefix, prefixAliasPath string
 	grouper := metric.NewSeriesGrouper()
 	timestamp := time.Unix(0, response.Update.Timestamp)
 	prefixTags := make(map[string]string)
 
+	// check if there are exceptions
+	if len(extension) != 0 {
```

Comment:
This check is not necessary as the loop will not be executed if `extensions` is empty.

Reply:
indeed. I removed it